README.md

```diff
@@ -22,13 +22,13 @@ Used by the [Dart Package site](https://pub.dartlang.org/).
 You can specify either a package (+ version) or a local directory to analyze:
 
 ```
-Usage: pana [<options>] <package> [<version>]
-       pana [<options>] --source path <directory>
+Usage: pana [<options>] <published package name> [<version>]
+       pana [<options>] --source path <local directory>
 
 Options:
       --flutter-sdk     The directory of the Flutter SDK.
   -j, --json            Output log items as JSON.
-  -s, --source          The source used to find the package.
+  -s, --source          The source where the package is located (hosted on http://pub.dartlang.org, or local directory path).
                         [hosted (default), path]
   
       --hosted-url      The server that hosts <package>.
```

Comment:
And its dependencies?

Reply:
I'm not sure, it will be up to the `pub` client to download them. If the `pubspec.yaml` is set up properly, they should be able to fetch them. We had a developer in the community who was using this, and AFAICT they got it to work for their case.